### ISSUE TITLE
versionType=2 is not a valid value for versionType. Try one of: Branch,Tag,Commit

### ISSUE THREAD

**Issue by hoovercj** [Microsoft/azure-devops-node-api#237]
1. Please check our current Issues to see if someone already reported this https://github.com/Microsoft/vsts-node-api/issues
2. Take a look at our Instructions for logging issues https://github.com/Microsoft/vsts-node-api/blob/master/CONTRIBUTING.md#instructions-for-logging-issues

### Environment
Node version: 8.10.0
Npm version: 5.6.0
OS and version: Win 10
azure-devops-node-api version: 6.6.0

### Issue Description

I'm getting the following exception:
```
{"$id":"1","innerException":null,"message":"versionType=2 is not a valid value for versionType. Try one of: Branch,Tag,Commit","typeName":"System.ArgumentException, mscorlib","typeKey":"ArgumentException","errorCode":0,"eventId":0}
```

Code used to get it is something like this:
```javascript
const gitVersion = {
    version: commitId,
    versionType: GitVersionType.Commit,
    versionOptions: GitVersionOptions.FirstParent,
}
const stream = await git.getItemContent(repositoryName, filePath, repositoryName, undefined, undefined, undefined, undefined, true, gitVersion);
console.log(stream.read().toString());
```

Basically I am using the npm package + the @types package and it doesn't like the versionType value.

**Comment by hoovercj** [comment]
Note: my repositoryName and projectName are the same so I use the same variable in both places but I'm aware that they are in fact two different parameters.

If I don't specify a version at all, I DO get the content of the file in master:

```javascript
const stream = await git.getItemContent(repositoryName, filePath, repositoryName);
console.log(stream.read().toString());
```

And if I omit the versionType and just provide `master` as the version, I get a different exception:

Code:
```javascript
const gitVersion: GitVersionDescriptor = {
    version: 'master',
}
const stream = await git.getItemContent(repositoryId, change.item.path, projectName, undefined, undefined, undefined, undefined, true, gitVersion);
console.log(stream.read().toString());
```


Exception:
```
{"$id":"1","innerException":null,"message":"TF401175:The version descriptor <Branch: version=master > could not be resolved to a version in the repository REPOSITORY_NAME","typeName":"Microsoft.TeamFoundation.Git.Server.GitUnresolvableToCommitException, Microsoft.TeamFoundation.Git.Server","typeKey":"GitUnresolvableToCommitException","errorCode":0,"eventId":3000}
```

**Comment by hoovercj** [comment]
To clarify, you mean `azure-devops-node-api`  6.6.2?

**Comment by damccorm** [comment]
Whoops yes, sorry

**Comment by hoovercj** [comment]
```
npm installnpm ERR! code ETARGET
npm ERR! notarget No matching version found for azure-devops-node-api@6.6.2
```

I'm assuming tt hasn't been published yet

**Comment by damccorm** [comment]
Ah, you're correct, sorry forgot I just had it locally - we'll try to get that out soon. I think once that's out this should probably be fixed. Is this blocking you?

**Comment by damccorm** [comment]
@bryanmacfarlane can we release today?

**Comment by hoovercj** [comment]
I am blocked, yes.

**Comment by hoovercj** [comment]
If you can't release today, maybe it's possible to send me a zip of the package (alias: cohoov)

**Comment by damccorm** [comment]
I just sent it, will follow up in this thread when the new version gets published

**Comment by FlorianRappl** [comment]
I am blocked as well. Would be nice to release 6.6.2 soon, as I don't want to fork / re-release the project internally (or copy it to our sources and change paths). Thanks!

**Comment by FlorianRappl** [comment]
Any news on this one? Thanks!

cc @damccorm
